src/spetlr/dbutils/FileExists.py

```diff
@@ -0,0 +1,29 @@
+from spetlr.functions import init_dbutils
+
+
+def file_exists(path: str) -> bool:
+    """
+    Helper function to check whether a file or folder exists.
+
+    Todo: Feel free to implement a better solution like the following:
+
+    This implementation should be more correct:
+
+    try:
+        init_dbutils().fs.ls(path)
+        return True
+    except Exception as e:
+        exc_info = sys.exc_info()
+        if "java.io.FileNotFoundException" in str(exc_info):
+            return False
+        else:
+            raise e
+
+
+    """
+
+    try:
+        init_dbutils().fs.ls(path)
+        return True
+    except Exception:
```

Comment:
Use the specific Exception related to the file not existing - it can be dangerous to just return `False` on all exceptions.

Reply:
I know - but see previous comment on @JeppeBlixen review. There is an unexpected bug when I create a more safe solution. Can you find the error? See e.g. this run: https://github.com/spetlr-org/spetlr/actions/runs/5355655874/jobs/9714198581